.github/workflows/ci-spring-boot-container-scan.yml

```diff
@@ -82,15 +82,19 @@ jobs:
           SBOM_NAME=$(echo ${{env.IMAGE-NAME}} | tr '/' '-')
           echo "SBOM_ARTIFACT_ID=$SBOM_NAME" >> "$GITHUB_OUTPUT"
 
-      - name: Create SBOM
-        uses: anchore/sbom-action@7ccf588e3cf3cc2611714c2eeae48550fbc17552 # pin@v0.15.11
-        with:
-          image: ${{env.IMAGE-NAME}}:${{env.IMAGETAG}}
-          artifact-name: sbom-${{steps.sbom-name.outputs.SBOM_ARTIFACT_ID}}-${{env.IMAGETAG}}.spdx
-
       - name: Run Trivy vulnerability scanner
         uses: aquasecurity/trivy-action@d710430a6722f083d3b36b8339ff66b32f22ee55 # pin@v0.19.0
```

Comment:
trur det hadde vore kjekt å kunne droppe å laste opp sbom artifact her i dei tilfella ein berre bygger på branchen, då er det unødvendig å laste opp den. Treng de berre på build-image. Men så er det kanskje i bruk av andre applikasjonar som ikkje er spring-boot og då er det brått kjekt å ha den sbom opplastinga her? Tenker default skrudd av stepet "upload trivy report as github artifact..." så har ein mogelegheit til å skrud det på manuelt?

Reply:
mogeleg du berre kan slette den utan mogelegheit for å skruv på. Denne skal vel berre vera i bruk frå spring-boot applikasjonar, og dei blir vel uansett bygga med den andre workflowen (over).